corehq/apps/ota/views.py

```diff
@@ -37,6 +38,7 @@
 from .utils import demo_user_restore_response, get_restore_user, is_permitted_to_restore, handle_401_response
 
 
+@location_safe
```

Comment:
do we need to `@location_safe` the other views in this file too? I think case claim won't work without them (this was on my backlog to do, so no worries if you don't want to add it here, I'll do it in a followup)